back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/stats_users_controller.rb

```diff
@@ -216,7 +216,7 @@ def users_by_custom_field
           @custom_field = CustomField.find(params[:custom_field_id])
           serie = users_by_custom_field_serie
           if %w[select multiselect].include?(@custom_field.input_type)
-            options = @custom_field.custom_field_options.select(:key, :title_multiloc)
+            options = @custom_field.custom_field_options.order(:ordering).select(:key, :title_multiloc)
```

Comment:
This fix is fine, but there's another approach we could take.

As we'll (almost) always will want to order the `custom_field_options` by the `ordering` attribute, we could also add it to the model. This is explained in the Rails guides [here](https://guides.rubyonrails.org/active_record_querying.html#applying-a-default-scope), and if you look for it in the codebase you'll find some examples too.

I think in this case, as it's something so universal, it makes more sense to solve it that way, though your solution is definitely not wrong, and often preferable.